src/routes/docs/products/auth/email-password/+page.markdoc

```diff
@@ -80,25 +74,28 @@ promise.then(function (response) {
 });
 ```
 
-# Login {% #login %}
+Next, implement the verification page in your app. This page will parse the secrets passed in through the `userId` and `secret` query parameters. In this example, the code below will be found in the page served at `https://example.com/verify`.
 
-After you've created your account, users can be logged in using the Create Email Session route.
+Since the secrets are passed in through url params, it will be easiest to perform this step in the browser.
 
 ```client-web
 import { Client, Account } from "appwrite";
 
 const client = new Client()
-    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
-    .setProject('<PROJECT_ID>');                 // Your project ID
+    .setProject('<PROJECT_ID>'); // Your project ID
 
 const account = new Account(client);
 
-const promise = account.createEmailPasswordSession('email@example.com', 'password');
+const urlParams = new URLSearchParams(window.location.search);
+const secret = urlParams.get('secret');
+const userId = urlParams.get('userId');
+
+const promise = account.updateVerification(userId, secret);
 
 promise.then(function (response) {
-    console.log(response); // Success
+    console.log(response);
 }, function (error) {
-    console.log(error); // Failure
+    console.log(error);
```

Comment:
For consistency, let's keep the Success and Failure comments for all of the snippets